### PR TITLE
Add yandex-beta

### PR DESCRIPTION
**Pull request by a-x-**
Yandex browser beta (also named as Custo) with absolutely new UI :)

### CODE REVIEW

**Comment by vitorgalvao** [comment]
If `Custo` is a name, then two `name` should be used. Also you, should add the vendor.

Like so:

```ruby
cask :v1 => 'yandex-beta' do
  version :latest
  sha256 :no_check

  url 'https://browser.yandex.ru/download/?betacusto=1'
  name 'Yandex.Browser'
  name 'Custo'
  homepage 'https://browser.yandex.ru/beta/'
  license :commercial
  tags :vendor => 'Yandex'

  app 'Yandex.app'
end
```

**Comment by a-x-** [comment]
fixed.
Am i should squash commits?

File: Casks/yandex-beta.rb [hunk 1]
```diff
@@ -0,0 +1,13 @@
+cask :v1 => 'yandex-beta' do
+  version :latest
+  sha256 :no_check
+
+  url 'https://browser.yandex.ru/download/?betacusto=1'
+  name 'Yandex browser beta'
```
Comment (vitorgalvao) [inline Casks/yandex-beta.rb]: `Yandex.Browser`

**Comment by vitorgalvao** [comment]
Yes.

**Comment by a-x-** [comment]
fixed

**Comment by a-x-** [comment]
And i uninstalled Custo and run `brew cask install yandex-beta` for test:

==> Downloading https://browser.yandex.ru/download/?betacusto=1&os=mac
######################################################################## 100,0%
==> Symlinking App 'Yandex.app' to '/Users/invntrm/Applications/Yandex.app'
🍺  yandex-beta staged at '/opt/homebrew-cask/Caskroom/yandex-beta/latest' (320 files, 197M)

**Pull request merged**